.github/workflows/e2e-test.yml

```diff
@@ -59,10 +60,7 @@ jobs:
             tezos-client --endpoint http://my-sandbox:20000 config update
             tezos-client import secret key alice unencrypted:edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq
             tezos-client import secret key bob unencrypted:edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt
-            cd dev
-            nohup python3 test-zeekoe.py merch-setup --url "http://my-sandbox:20000" -v &
-            nohup python3 test-zeekoe.py cust-setup --url "http://my-sandbox:20000" -v &
-            python3 test-zeekoe.py test-all
+            cargo test --all-features --all-targets --verbose
```

Comment:
Can we do something with the command so it only runs the integration tests, it runs unit tests as well?

Reply:
Good question. We can do
```
cargo test --all-features --tests integration_tests
```
Then we can change the pull request yml to do
```
cargo test --all-features --lib --bins --examples --benches
```
However, these two commands would exclude any new integration tests we add (e.g. in `test/`). Is this worth being concerned about? Presumably if we write new tests, we will pay attention to whether they're actually getting run.

I have not found a way to tell cargo to ignore / skip `integration_tests` but run everything else.